src/Microsoft.ML.Data/Evaluators/MamlEvaluator.cs

```diff
@@ -127,11 +127,11 @@ protected MamlEvaluatorBase(ArgumentsBase args, IHostEnvironment env, string sco
 
             // Get the label column information.
             string lab = EvaluateUtils.GetColName(LabelCol, schema.Label, DefaultColumnNames.Label);
```

Comment:
>string lab [](start = 12, length = 10)

can we change it to var label?
it just looks odd to have lab and then few lines later weight.

Reply:
Mind if I go the other way? I like to be explicit with my types (in this case not very obvious) unless it's something longer, or the type is immediately obvious to a reader. (Except `int`... don't get people that aren't explicit about `int`.)

---
In reply to: [200458796](https://github.com/dotnet/machinelearning/pull/496#discussion_r200458796) [](ancestors = 200458796)

Reply:
As you can probably see from my comment, I tend to prefer var, but i don't have problem with explicit types, it was more about shortening label to lab, which for some reason bugs me enormously.

---
In reply to: [200460563](https://github.com/dotnet/machinelearning/pull/496#discussion_r200460563) [](ancestors = 200460563,200458796)